### DISCUSSION TITLE
Nie mogę wystawić oferty: komnikat - Zablokowana sprzedaż (https://salescenter.allegro.com.allegrosandbox.pl)

### DISCUSSION THREAD

**Discussion by ryszardlet** [allegro/allegro-api#9953] [Suggestion - verification]
j.w. 
W sandboxie nie mogę wystawić oferty:

_"Zablokowana sprzedaż
Sprzedaż na Twoim koncie jest obecnie zablokowana. Powodem mogą być nieopłacone rachunki lub inne naruszenia Regulaminu. Aby odblokować możliwość sprzedaży, ureguluj zaległe płatności lub skontaktuj się z nami."_

Czy mogę prosić o odblokowanie konta? Dane konta to:
marketplaceLogin: hummerce
sellerId: 106855400

**Comment by MaciejFrackowiak** [comment]
Zakładka Discussions służy do zgłaszania sugestii. Twój wątek przeniosłem do zakładki Issues - https://github.com/allegro/allegro-api/issues/9956.
